### PR TITLE
Create nested pages

### PR DESCRIPTION
**Pull request by hannahkamundson**
new pull request. lord be with us

### CODE REVIEW

**Comment by hannahkamundson** [comment]
PS only look at commits from today if possible

File: frontend/src/pages/Table/District/Districts.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import { getDistricts } from 'services/districtServices';
+import PropTypes from 'prop-types';
+
+import { withStyles,
+} from '@material-ui/core';
+
+import { loadingJSX } from 'components/sharedStyles/LoadingStyles';
+import { TablePageStyles } from 'components/sharedStyles/Table/TablePageStyles';
+import { CreateDistrictTable } from 'components/sharedStyles/Table/CreateTablesStyle';
```
Comment (HTempleman) [inline frontend/src/pages/Table/District/Districts.js]: Yay!

**Comment by hannahkamundson** [comment]
@HTempleman Okay finally got it to work! I will look into a tool that automatically imports. 
All I had to do was delete some of the old stuff that I somehow hadn't deleted.

**Comment by hannahkamundson** [comment]
@HTempleman to check you can look at the following urls
localhost:8000/student
localhost:8000/student/1
localhost:8000/school
localhost:8000/school/1
localhost:8000/program
localhost:8000/program/1
localhost:8000/district
localhost:8000/district/1
localhost:8000/course
localhost:8000/course/1

**Comment by HTempleman** [comment]
Got it working. This is a vast improvement over what we had! We'll want to generalize things further, but we have a good foundation to spring from now. Great job Hannah.

**Pull request merged**